src/express.re

```diff
@@ -115,10 +115,18 @@ module Middleware = {
 module MakeBindFunctions (T: {type t;}) => {
   external use : T.t => Middleware.t => unit = "" [@@bs.send];
   external useN : T.t => array Middleware.t => unit = "use" [@@bs.send];
-  external useOnPath : T.t => path::string => Middleware.t => unit = "use" [@@bs.send];
-  external useOnPathN: T.t => path::string => array Middleware.t => unit = "use" [@@bs.send];
-  external get : T.t => path::string => Middleware.t => unit = "" [@@bs.send];
-  external getN: T.t => path::string => array Middleware.t => unit = "" [@@bs.send];
+  external useOnPath : T.t => path::string => Middleware.t => unit 
+    = "use" [@@bs.send];
+  external useOnPathN: T.t => path::string => array Middleware.t => unit 
+    = "use" [@@bs.send];
+  external get : T.t => path::string => Middleware.t => unit 
+    = "" [@@bs.send];
+  external getMany: T.t => path::string => array Middleware.t => unit 
```

Comment:
I'm also thinking `getMany` is a bit confusing, it indicates the function retrieves many somethings, rather than _with_ many something. So perhaps `WithMany` is a better suffix.